pkg/interceptor.go

```diff
@@ -9,8 +9,10 @@ type (
 		Service string
 		// Name of method handling request
 		Method string
-		// Endpoint of request, can be used for retrying
-		Endpoint Endpoint
+		// Send is true if the request is a streaming Send
```

Comment:
What's the rationale for these two fields? I could see `Send` helping differentiate between the initial request and the streaming request in the case of websocket but not sure I understand the point of `Recv`?

Reply:
Got it, makes sense. I wonder if we could have only one field then?

Reply:
I suppose we could replace it with an enum.

Reply:
Maybe something like: `goa.InterceptorMethod`, `goa.InterceptorStreamingSend`, `goa.InterceptorStreamingRecv`.

Reply:
Yes, `goa.InterceptorUnary` instead of `Method` maybe?